packages/volto/src/components/manage/Blocks/Block/__snapshots__/Settings.test.jsx.snap

```diff
@@ -3,15 +3,18 @@
 exports[`Settings renders block settings form without schema enhancers 1`] = `
 <div>
   <div
-    class="ui form"
-  >
-    <div
-      id="blockform-fieldset-default"
-    >
-      <div
-        class="ui segment form attached"
-      />
-    </div>
-  </div>
+    data-schema="{
+  \\"fieldsets\\": [
+    {
+      \\"title\\": \\"Default\\",
+      \\"id\\": \\"default\\",
+      \\"fields\\": []
+    }
+  ],
+  \\"properties\\": {},
+  \\"required\\": []
+}"
```

Comment:
It's ugly, but valid.

The lazy-loaded components, when used nested inside another component, give us more challenges when rendering for unit tests. Since they are unit tests, we don't really need to test nested components, because they have their own tests, so I worked around the rendering issues by mocking the lazy-loaded components and making it so that we just render the props that are passed to them. In other words, we don't render nested forms, but just the fact that we are rendering a form with the given schema.

The ugly syntax that showed up is how a json schema is rendered with snapshot testing.